hicommonwealth/commonwealth
# Unified token creation flow

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/10243

## Description of Changes
We had 2 token creation flows, a quick one with a single step and another one with 3 individual steps. To reduce friction, the user flow with more steps is now removed to match the quick token launch flow.

## "How We Fixed It"
N/A

## Test Plan
- Click on the "Launch Token" button from the sidebar
- Attempt to launch a token, and verify the steps/flow is same to the launch token flow from `/communities` => `Launch token` cta
- Verify no regressions occured

## Deployment Plan
N/A

## Other Considerations
N/A
